public/app/features/alerting/unified/components/contact-points/useContactPoints.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * This hook will combine data from both the Alertmanager config
+ * and (if available) it will also fetch the status from the Grafana Managed status endpoint
```

Comment:
Nice!

Reply:
I actually have this working in a separate branch – using RTK Query and a few other nice things to keep the UI state up-to-date. Stay tuned!